src/layouts/styles.js

```diff
@@ -2,15 +2,50 @@ import facepaint from 'facepaint'
 const breakpoints = [576, 768, 1024, 1200]
 const mq = facepaint(breakpoints.map(bp => `@media (min-width: ${bp}px)`))
 
-export const styles = {
-  pageWrapper: mq({
-    overflow: 'auto',
-    width: '100%',
-    height: 'auto',
-    display: 'block',
-    position: 'relative',
-    paddingLeft: ['32px', '32px', '80px', '135px', '160px'],
-    paddingRight: ['32px', '32px', '80px', '135px', '160px'],
-    paddingBottom: ['60px', '60px', '60px', '226px'],
-  }),
-}
+export const pageWrapper = mq({
+  overflow: 'auto',
+  width: '100%',
+  height: 'auto',
+  display: 'block',
+  position: 'relative',
+  paddingLeft: ['32px', '32px', '80px', '135px', '160px'],
+  paddingRight: ['32px', '32px', '80px', '135px', '160px'],
+})
+
+// ############### STYLING FOR ABOUT INDEX PAGE #################
+
+export const aboutWrapper = mq({
```

Comment:
@Alphascythian Let's copy the way we implement components file structure and create a folder `About` with the `styles.js` and `index.js`